src/WingetCreateCLI/Commands/BaseCommand.cs

```diff
@@ -578,7 +583,40 @@ protected async Task<bool> GitHubSubmitManifests(Manifests manifests, string prT
             }
 
             return true;
-        }
+        }
+
+        /// <summary>
+        /// Returns the title to be used for the pull request.
+        /// </summary>
+        /// <param name="currentManifest">Manifest object containing metadata of new manifest to be submitted.</param>
+        /// <param name="repositoryManifest">Manifest object representing an already exisitng manifest in the repository.</param>
+        /// <returns>A string representing the pull request title.</returns>
+        protected string GetPRTitle(Manifests currentManifest, Manifests repositoryManifest = null)
+        {
+            // Use custom PR title if provided by the user.
+            if (!string.IsNullOrEmpty(this.PRTitle))
+            {
+                return this.PRTitle;
+            }
+
+            string packageId = currentManifest.SingletonManifest != null ? currentManifest.SingletonManifest.PackageIdentifier : currentManifest.InstallerManifest.PackageIdentifier;
+            string currentVersion = currentManifest.SingletonManifest != null ? currentManifest.SingletonManifest.PackageVersion : currentManifest.InstallerManifest.PackageVersion;

```

Comment:
Since we don't allow singleton manifests, we should prioritize the version manifest when retrieving these values before resorting to singleton (In theory we should never have to use singleton manifest)

```
string packageId = currentManifest.VersionManifest != null ? currentManifest.Version.PackageIdentifier : currentManifest.SingletonManifest.PackageIdentifier;
```

Reply:
Same for `currentVersion`

Reply:
Makes sense; Updated to prioritize multi-manifest.